src/_internal/hangul.ts

```diff
@@ -12,6 +12,15 @@ export function isHangulAlphabet(character: string) {
   return /^[ㄱ-ㅣ]$/.test(character);
 }
 
+/**
+ * @name isHangulText
+ * @description
+ * text를 받으면 해당 text가 한글 문장인지를 구분합니다.
+ *
+ */
+export function isHangulText(text: string): boolean {
```

Comment:
요렇게 조금 더 명확히 한글만 있는가를 검증하는것을 나타내는것은 어떤가요?

```suggestion
export function isHangulOnly(text: string): boolean {
```

Reply:
사용하는쪽도 수정이 되면 좋겠습니다~ 감사합니다 🙇